### PR TITLE
chore: use new-ci to replace the old ci

### PR DESCRIPTION
**Pull request by sailorlqh**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,22 +1,52 @@
...
+# test implementation has finished
 on:
-  push:
-    branches:
-      - 'main'
-  pull_request:
-    branches:
-      - 'main'
   workflow_dispatch:
-  workflow_call:
```
Comment (yolocs) [inline .github/workflows/ci.yml:12]: I think we would want to keep this?

Comment (sailorlqh) [reply .github/workflows/ci.yml]: Sorry for the miss, fixed

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,22 +1,52 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# We will remove ci.yml and rename this to ci.yml
+# after the new integration test implementation
+# has finished.
```
[the remark this answers is not in the document]
Comment (sailorlqh) [reply .github/workflows/ci.yml:18]: Done

[the remark this answers is not in the document]
Comment (yolocs) [reply .github/workflows/ci.yml:18]: Still not removed?

**Review by yolocs — CHANGES_REQUESTED** [review]

File: test/integ/main_test.go [hunk 3]
```diff
@@ -13,353 +13,246 @@
 // limitations under the License.
 
 // Main entry point for integration tests.
-package integ
+package integration
```
Comment (sqin2019) [inline test/integ/main_test.go:16]: This is really difficult to read, to double confirm, this is just renaming the main_new_test.go?

**Review by sqin2019 — APPROVED** [review]

File: test/integ/main_test.go (hunk 3 above)
Comment (sailorlqh) [reply test/integ/main_test.go:16]: Yes, we removed the old ci, and replaced it with the new ci, the old ci is in package `integ`, the new ci is in package `integration`

**Review by yolocs — APPROVED** [review]

**Pull request merged**